src/webgpu/web_platform/canvas/readbackFromWebGPUCanvas.spec.ts

```diff
@@ -329,3 +329,33 @@ g.test('drawTo2DCanvas')
 
     readPixelsFrom2DCanvasAndCompare(t, ctx, expect[t.params.alphaMode]);
   });
+
+g.test('transferToImageBitmap_unconfigured_nonzero_size')
+  .desc(
+    `Regression test for a crash when calling transferImageBitmap on an unconfigured. Case where the canvas is not empty`
+  )
+  .fn(t => {
+    const canvas = createCanvas(t, 'offscreen', 2, 3);
+    canvas.getContext('webgpu');
+
+    // Transferring gives an ImageBitmap of the correct size filled with transparent black.
+    const ib = canvas.transferToImageBitmap();
+    t.expect(ib.width === 2);
+    t.expect(ib.height === 3);
+
+    // TODO check the pixels are black somehow
+  });
+
+g.test('transferToImageBitmap_unconfigured_zero_size')
```

Comment:
Should fail regardless of whether it's configured. Maybe add a true-false param for whether to configure it before transferToImageBitmap?